build_scripts/libtock_layout.ld

```diff
@@ -116,6 +116,10 @@ SECTIONS {
         *(.rodata.*)
     } > FLASH
 
+    ..data.rel.ro.local ALIGN(4) : {
+        *(.data.rel.ro.local.*)
+    } > FLASH
```

Comment:
Stylistically, my preference is to include `.data.rel.ro` into `.rodata` rather than putting it into its own section. I.e.:

```
    .rodata ALIGN(4) : {
        *(.rodata.*)

        /* Generated by GCC in libraries for variables that still have relocations but are constant
         * at runtime. See https://www.airs.com/blog/archives/189 for background. */
        *(.data.rel.ro.*) 
    }
```

Reply:
sweet, I added this change!